### DISCUSSION TITLE
does otel-contrib support https?

### DISCUSSION THREAD

**Discussion by isshwar** [open-telemetry/opentelemetry-collector-contrib#17493] [Q&A]
Hi,

I am planning to run otel-contrib in a kubernetes cluster and want to run it behind load balancer. This results in otel-contrib at https endpoint. would otel-contrib be run at https?

thanks

**Comment by jpkrohling** [comment]
Which component are you interested? All receivers using HTTP Server Settings are able to listen on TLS ports (see https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/confighttp#server-configuration).
